grafana/resource_data_source.go

```diff
@@ -461,8 +461,8 @@ source selected (via the 'type' argument).
 				Type:        schema.TypeList,
 				Optional:    true,
 				Sensitive:   true,
-				Description: "Deprecated: Use secure_json_data instead. It supports arbitrary JSON data, and therefore all attributes.",
-				Deprecated:  "Use secure_json_data instead. It supports arbitrary JSON data, and therefore all attributes.",
+				Description: "Deprecated: Use secure_json_data_map instead. It supports arbitrary JSON data, and therefore all attributes.",
+				Deprecated:  "Use secure_json_data_map instead. It supports arbitrary JSON data, and therefore all attributes.",
```

Comment:
```suggestion
				Description: "Deprecated: Use secure_json_data_encoded instead. It supports arbitrary JSON data, and therefore all attributes.",
				Deprecated:  "Use secure_json_data_encoded instead. It supports arbitrary JSON data, and therefore all attributes.",
```

Reply:
Just checked the [documentation](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/data_source#secure_json_data_encoded) (should have done that earlier I guess, sorry. Will update the PR.